master/internal/core_experiment.go

```diff
@@ -290,9 +290,7 @@ func (m *Master) patchExperiment(c echo.Context) (interface{}, error) {
 		dbExp.Config.SetResources(resources)
 	}
 	if patch.Description != nil {
```

Comment:
do you know if anything still calls this echo patch?

Reply:
our cli does. I moved our cli usage for description and name to use the new API. 
once we add support for what 

```
		Resources *struct {
			MaxSlots api.MaybeInt `json:"max_slots"`
			Weight   *float64     `json:"weight"`
		} `json:"resources"`
		CheckpointStorage *struct {
			SaveExperimentBest int `json:"save_experiment_best"`
			SaveTrialBest      int `json:"save_trial_best"`
			SaveTrialLatest    int `json:"save_trial_latest"`
		} `json:"checkpoint_storage"`
```
is offering in our new apis we should be able to move off of this.